task/orka-teardown/0.1/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+${KUBECTL_CMD} delete clusterrole orka-runner --ignore-not-found
+${KUBECTL_CMD} delete clusterrolebinding orka-runner --ignore-not-found
```

Comment:
do we need this now? :thinking:  or probably we can rename these also

Reply:
No longer needed - removed.